data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,919 @@
+
+
+outfit "Fleck"
+	weapon
+		sprite "projectile/fleck"
+			"frame rate" 6
+			"random start frame"
+			"rewind"
+		"hit effect" "bombardment impact"
+		"lifetime" 30
+		"random lifetime" 6
+		"inaccuracy" 0.3
+		"shield damage" 72
+		"hull damage" 150
+		"hit force" 60
+		"missile strength" 3
+
+
+effect "fleck fire"
+	sprite "effect/emp torpedo fire"
+		"scale" 1
+		"random start frame"
+		"frame rate" 60
+	"lifetime" 3
+	"random angle" 30
+
+
+# Energy Weapons:
+
+outfit "Phased Array Laser (Green)"
+	plural "Phased Array Lasers (Green)"
+	category "Turrets"
+	licenses
+		"Avgi Civilian"
+	cost 120000
+	thumbnail "outfit/green phaser"
+	"mass" 6
+	"outfit space" -6
+	"weapon capacity" -6
+	"turret mounts" -1
+	"optical jamming" 9
+	"lasing power" -2
+	weapon
+		"hardpoint sprite" "hardpoint/green phaser"
+		"die effect" "laser ejecta" 4
+		"die effect" "green phaser reflection" 30
+		"die effect" "green phaser scattering"
+		"fire effect" "green phaser fire"
+		"anti-missile" 3
+		"velocity" 1200
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 12
+		"firing heat" 9
+	description "This phased array laser, or phaser, binds thousands of individual optical fibers together to create a single collimated beam precise enough to target projectiles in flight. A verdant glow strong enough to blind the naked eye is the only indication of this weapon firing."
+	description "The turret itself only contains the optics needed to synthesize and target the beam, allowing the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support three Green Phasers."
+
+
+outfit "Phased Array Laser (Blue)"
+	plural "Phased Array Lasers (Blue)"
+	category "Turrets"
+	licenses
+		"Avgi Military"
+	cost 120000
+	thumbnail "outfit/blue phaser"
+	"mass" 9
+	"outfit space" -9
+	"weapon capacity" -9
+	"turret mounts" -1
+	"optical jamming" 12
+	"lasing power" -3
+	weapon
+		"hardpoint sprite" "hardpoint/blue phaser"
+		sprite "projectile/avgi phaser blue"
+			"frame rate" 1
+		"hit effect" "laser ejecta" 4
+		"hit effect" "blue phaser reflection" 30
+		"hit effect" "blue phaser scattering"
+		"fire effect" "blue phaser fire"
+		"velocity" 3600
+		"lifetime" 1
+		"reload" 1
+		"turret turn" 6
+		"firing energy" 12
+		"firing heat" 12
+		"shield damage" 3
+		"hull damage" 4.2
+		"heat damage" 6
+		"damage dropoff" 0 3600
+	description "This phased array laser, or phaser, binds thousands of individual optical fibers together to create a single collimated beam precise enough to target projectiles in flight. An azure glow strong enough to blind the naked eye is the only indication of this weapon firing."
```

Comment:
You should focus more on how this version of the weapon targets ships instead of acting as anti-missile.